themes/default/content/blog/examining-recommended-practices-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Examining Recommended Practices: Code Placement and Stacks"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2023-02-15T13:01:26-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: This is the first in a series of blog posts on recommended practices for using Pulumi. Code placement and the use of Pulumi stacks are the focus areas.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - development-environment
+    - cloud-engineering
```

Comment:
Suggest removing `development-environment` since that isn't really discussed here, and adding `zephyr` to make it easy to group these up later.

```suggestion
    - zephyr
    - best-practices
    - cloud-engineering
```

Reply:
Fixed in 3b48f0ca